roles/core/nfs_server/tasks/main.yml

```diff
@@ -20,6 +20,12 @@
 
 - meta: flush_handlers
 
+- name: Create directory                                                                                                                                                                                             file:
+    path: "{{ nfs[item]['export'] }}"
```

Comment:
I guess it should be nfs[item]['mount'] which is the path the NFS server should export?